### PR TITLE
Discounts: Removed java.util.date and start using LocalDate and LocalTime

### PR DESCRIPTION
**Pull request by Tat0**

### CODE REVIEW

File: src/main/webapp/resources/js/adminJs/adm-discount.js [hunk 1]
```diff
@@ -77,8 +77,10 @@ function addNewDiscount(method,bool){
   var inputData = {
     reason: $("#DReason").val(),
     value : $("#DValue").val(),
-    startDate :startDate,
-    endDate : endDate,
+    startDate : "23-03-2017",
+    endDate : "23-03-2017",
+    startTime: "11:00",
+    endTime: "23:00",
```
Comment (Tat0) [inline src/main/webapp/resources/js/adminJs/adm-discount.js]: It's just for the test. You don't have to write smth like "Magic numbers, please remove it". I will remove it all  when Yevgen will finish JQuery tables

File: src/main/java/ua/softserveinc/tc/controller/admin/restful/AdminDiscountController.java [hunk 2]
```diff
@@ -37,15 +38,14 @@ public DayDiscountDTO getDayDiscountById(@PathVariable long id) {
...
-  @GetMapping("/{startDate}/{endDate}")
-  public ResponseEntity<String> checkUniqueOfDayDiscount(@PathVariable long startDate,
-      @PathVariable long endDate) {
-    if (dayDiscountService.getDayDiscountsForPeriod(new Date(startDate), new Date(endDate)).size()
-        > 0) {
-      return new ResponseEntity<String>("No ok", HttpStatus.OK);
-    } else {
-      return new ResponseEntity<String>("ok", HttpStatus.OK);
-    }
+  @GetMapping("/{startDate}/{endDate}/{startTime}/{endTime}")
```
Comment (Yevhensh) [inline src/main/java/ua/softserveinc/tc/controller/admin/restful/AdminDiscountController.java]: Better use RequestParam rather than so much PathVariables

Comment (Tat0) [inline src/main/java/ua/softserveinc/tc/controller/admin/restful/AdminDiscountController.java]: http://stackoverflow.com/questions/13715811/requestparam-vs-pathvariable

And @RequestParam isn't required.

Comment (Yevhensh) [inline src/main/java/ua/softserveinc/tc/controller/admin/restful/AdminDiscountController.java]: for example @GetMapping("/discounts") and then fill in with RequestParam like:
../discounts?startDate=5&endDate=10...
this state do not respresent your data 
http://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api

Comment (Yevhensh) [inline src/main/java/ua/softserveinc/tc/controller/admin/restful/AdminDiscountController.java]: or /adm-discounts/{startDate}/{endDate}...

Comment (Yevhensh) [inline src/main/java/ua/softserveinc/tc/controller/admin/restful/AdminDiscountController.java]: okay) convinced

**Pull request merged**